lib/encoding.js

```diff
@@ -11,8 +11,7 @@
   // If we're in node require encoding-indexes and attach it to the global.
   if (typeof module !== "undefined" && module.exports &&
     !global["encoding-indexes"]) {
-    global["encoding-indexes"] =
-      require("./encoding-indexes.js")["encoding-indexes"];
+      require("./encoding-indexes.js");
```

Comment:
This directly conflicts with the change made for b4e5bc26e26e51f56e3daa9f13138c79f49d3c34 for #67